### PR TITLE
Added MD file to design folder describing the design of the About plugin

### PR DESCRIPTION
**Pull request by matt-bowers**
Hey all, added a quick Markdown file to @ellisonbg's [design directory](https://github.com/jupyter/jupyterlab/tree/master/design) to talk about the possible future of the About plugin!

### CODE REVIEW

File: design/about.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+This document describes the design of the "About JupyterLab" plugin. This document will illustrate research done using Personas from users of the Jupyter Notebook, as well as describe all of the tasks and operations that the user should be able to do with this design, as well as how our design will look visually.
+
+## Personas
+
+### Ricardo Godfrey 
+
+* Male 25, Full Stack Engineer
+
+* uses single notebook to explain a very specific topic or teach people something
```
Comment (blink1073) [inline design/about.md]: Uses*

**Comment by blink1073** [comment]
Looks great, @matt-bowers, just one minor formatting comment.

**Comment by blink1073** [comment]
Thanks!

**Comment by matt-bowers** [comment]
Thank you! 😄

**Pull request merged**